### PR TITLE
[impl] account setup during initialization

### LINKED ISSUE #3: Basic private key and account management
We should implement some basic key/account management logic.

- [x] Key creation (recovery phrase? test accounts probably don't need password protection)
- [ ] Account switching
- [x] Key/account naming and metadata so it makes sense

The next stage would be to allow collaboration and be a good local keychain/wallet.

What are some useful functions for storing/accessing keys?

### LINKED ISSUE #9: Command: double init
`double init` should initialize a double environment in the current directory. It should:

* Create a `double.yaml` if there is not already one. If there is, use it.
* The newly created config file should contain values inherited from the global config.
* Genesis block and stuff should be created.
* If there already is data, we should see if it's what the genesis config wants (account and stuff). If not, we ask user whether it should be nuked so we can reset. If it is, nothing needs to be done.
* Finally we should report state to user and let the user run `double start` to start the local chain (and `double connect`, etc. for remote chain)

### PR DESCRIPTION
**Pull request by danqing**
Closes #3. Closes #9.

### CODE REVIEW

**Comment by danqing** [comment]
Ready for review @hengchu.

File: src/backend/ethereum/index.ts [hunk 1]
```diff
@@ -14,16 +13,32 @@ import Geth from './geth';
...
+  const genesis = JSON.parse(
+    fs.readFileSync(ETHEREUM_PROJECT_GENESIS).toString(),
+  );
+  for (const key of Object.keys(accounts).sort()) {
+    genesis.alloc[accounts[key]] = {balance: '10000000000000000000'};
+  }
+
   fs.ensureDirSync(datadir);
-  fs.copySync(ETHEREUM_PROJECT_GENESIS, file);
+  fs.writeFileSync(file, JSON.stringify(genesis), 'utf8');
```
Comment (hengchu) [inline src/backend/ethereum/index.ts]: Is the encoding parameter necessary?

**Comment by hengchu** [comment]
LGTM! But reading the changes reminds me that we might want to test utf-8 support with user inputs thoroughly eventually. Users might want to include them in the configuration files, for example.

File: src/backend/ethereum/index.ts (hunk 1 above)
Comment (danqing) [reply src/backend/ethereum/index.ts]: switched to a better helper method.

**Comment by danqing** [comment]
Yeah fair point.

**Pull request merged**